main.tf

```diff
@@ -117,8 +117,13 @@ resource "aws_elasticache_replication_group" "default" {
 # CloudWatch Resources
 #
 resource "aws_cloudwatch_metric_alarm" "cache_cpu" {
-  count               = var.enabled ? 1 : 0
-  alarm_name          = "${module.label.id}-cpu-utilization"
+  count = (
```

Comment:
Sorry I didn't think of this earlier but you could use the `range` function and do nested loops the way we do in DynamoDB.
```terraform
for i in range(var. cluster_mode_replicas_per_node_group): [
  for j in range(var. cluster_mode_num_node_groups): [
    # Construct alarm configuration here.

  ]
]
```

And then you can put all of this into locals as well.

Reply:
oh wow this is wayyyy nicer